contracts/modules/staking/StakingPool.sol

```diff
@@ -849,11 +849,11 @@ contract StakingPool is IStakingPool, Multicall {
   }
 
   function getTrancheCapacities(
-    uint productId,
     uint firstTrancheId,
     uint trancheCount,
     uint capacityRatio,
-    uint reductionRatio
+    uint reductionRatio,
+    uint productTargetWeight
```

Comment:
reason for doing this:

Couldn't find a way to reliably reverse the calculation in StakingProducts in the case that productTargetWeight is set to 0 (information loss).

We could still expose the same function (exact same interface) without this parameter and have an internal _ getTrancheCapacities that has the parameter and is used by all other ones.

I didn't see getTrancheCapacities used anywhere yet though so i modified it directly.